config/v1/types_infrastructure.go

```diff
@@ -562,6 +616,14 @@ type BareMetalPlatformStatus struct {
 	// datacenter DNS, a DNS service is hosted as a static pod to serve those hostnames
 	// to the nodes in the cluster.
 	NodeDNSIP string `json:"nodeDNSIP,omitempty"`
+
+	// loadBalancer defines how the load balancer used by the cluster is configured.
+	// +default={"openShiftManaged": {"type": "Default"}, "type": "OpenShiftManaged"}
+	// +kubebuilder:default={openShiftManaged: {type: Default}, type: OpenShiftManaged}
```

Comment:
Any idea why these are quoted differently?

Reply:
openapi requires the quotes, kubebuilder doesn't required but it works if you add them, so I'll do it for consistency.